docs/guides/rookieguide.md

```diff
@@ -1,7 +1,126 @@
+Poradnik został opracowany z myślą o graczach dopiero rozpoczynających członkostwo w naszej grupie - jego zamiarem jest poprowadzenie was przez rękę przez wszystkie meandry konfigurowania Army i radzenia sobie z wypracowanymi wewnątrz grupy systemami, do których należą między innymi zasady zapisywania się na misję. Poszczególne kroki są dokładnie opisane, te bardziej wymagające popieramy zrzutami ekranów, znajdziecie w nim również podstawowe FAQ dotyczące zagadnień ogólnych. W razie jakichkolwiek problemów ze zrozumieniem treści poradnika kontaktujcie się ze swoimi opiekunami Rekruterami.
 
-[Przejdź do poradnika dla rekrutów](https://docs.google.com/document/d/1nOPqnTeu9Flph8oBtN2FFMQ5dUvwFet_Y80uH1ckp1s/edit?usp=sharing)
+## *Discord Arma Forces*
+
+### 1. Wchodzimy na Discord Arma Forces.
+
+
+### 2. Wchodzimy na kanał #rekrut_game_annoucments
+
+
+
+Obrazek orientacyjny:
+
+
+![Obrazek orientacyjny](../_data/guides/rookie/rookie1.png)
+
+
+### 3. Na tym kanale znajdują się zapisy na misje ze slotami dla rekrutów. Zapisy działają na zasadzie odklikania reakcji, zasady listuje rekruter wysyłający ogłoszenie. 
+
+Przykładowe zapisy dla rekrutów:
+
+![Przykładowe zapisy](../_data/guides/rookie/rookie2.png)
+
+### Misje zawsze odbywają się o 20 chyba, że zostanie inaczej powiedziane.
+
+
+## Pobieranie modów
+
+### 1. Pobieranie modów.
+
+Klikamy na podany w zapisach link do modów, co przeniesie nas to na stronę z modlistami. Modlisty składają się z dwóch części: modów wymaganych, które potrzebne są do wejścia na serwer gdy dana modlista jest załadowana oraz modów opcjonalnych, które nie są potrzebne by rozegrać misje, lecz zawierają różnego rodzaju mody na lepsze dźwięki, dodatkowe ustawienia graficzne i tym podobne. Jesteście samemu w stanie zaznaczyć, które mody opcjonalne chcecie pobrać wraz z modlistą. Oczywiście możecie je, także w pełni zignorować, ale unikajcie dodawania ich wszystkich ponieważ część z nich się wyklucza lub w połączeniu może powodować różne błędy. Większość modów posiada krótki opis, kiedy na nie najedziecie, z którego możecie skorzystać w razie wątpliwości. Kiedy już dokonaliście wyboru klikamy przycisk "Pobierz/Download" i czekamy aż modlista się pobierze.
+
+### 2. Importowanie modlisty
+
+Po pobraniu modlisty wybieramy, w którym pliku chcemy ją zapisać. Następnie odpalamy launcher army i wybieramy zakładkę "mody/mods". Na zaimportowanie modlisty są dwa sposoby:
+1. W zakładce mody klikamy: więcej/more po czym importuj listę modów z.../import list of mods from a file... a następnie wybieramy zapisaną modlistę.
+2. Po wybraniu zakładki mody zapisaną modlistą przenosimy na launcher i puszczamy. 
+
+W prawym górnym rogu widzimi nazwę aktualnie załadowanej modlisty. Jeśli na nią klikniemy pokaże nam się lista wszystkich zaimportowanych modlist.
+
+**Kwestia Teamspeaka i ACRE.**
```

Comment:
I think Teamspeak and ACRE is not a matter of `2. Importowanie modlist`. Please include separate step for launching game and joining server.

Reply:
Resolved